projects/1.16/assets/wonderful-enchantments/wonderful_enchantments/lang/zh_cn.json

```
@@ -1,66 +1,98 @@
+	"enchantment.wonderful_enchantments.elder_guardian_favor.desc": "攻击命中后，用无法躲避或阻挡的激光束攻击敌人。",
+	"enchantment.wonderful_enchantments.elder_guardian_favor.description": "攻击命中后，用无法躲避或阻挡的激光束攻击敌人。",
+	"enchantment.wonderful_enchantments.harvester": "收割者",
+	"enchantment.wonderful_enchantments.harvester.desc": "右击收割作物时，有概率加速附近作物的生长。",
+	"enchantment.wonderful_enchantments.harvester.description": "右击收割作物时，有概率加速附近作物的生长。",
+	"enchantment.wonderful_enchantments.mithridatism": "抗毒",
+	"enchantment.wonderful_enchantments.mithridatism.desc": "减少部分负面效果的伤害（中毒，凋零等），并给予伤害吸收效果。",
+	"enchantment.wonderful_enchantments.mithridatism.description": "减少部分负面效果的伤害（中毒，凋零等），并给予伤害吸收效果。",
+	"effect.wonderful_enchantments.mithridatism_protection": "抗毒",
+	"wonderful_enchantments.mithridatism_level_up": "你对负面效果有了更强的抵抗力！",
+	"enchantment.wonderful_enchantments.fuse_cutter": "熔断器",
+	"enchantment.wonderful_enchantments.fuse_cutter.desc": "使附近的爆炸无效化。",
+	"enchantment.wonderful_enchantments.fuse_cutter.description": "使附近的爆炸无效化。",
+	"enchantment.wonderful_enchantments.sixth_sense": "第六感",
+	"enchantment.wonderful_enchantments.sixth_sense.desc": "静止片刻后高亮显示附近的敌对生物。",
+	"enchantment.wonderful_enchantments.sixth_sense.description": "静止片刻后高亮显示附近的敌对生物。",
+	"enchantment.wonderful_enchantments.slowness_curse": "缓慢诅咒",
+	"enchantment.wonderful_enchantments.slowness_curse.desc": "降低玩家的移动速度。",
+	"enchantment.wonderful_enchantments.slowness_curse.description": "降低玩家的移动速度。",
+	"enchantment.wonderful_enchantments.fatigue_curse": "疲劳诅咒",
+	"enchantment.wonderful_enchantments.fatigue_curse.desc": "降低玩家的挖掘速度。",
+	"enchantment.wonderful_enchantments.fatigue_curse.description": "降低玩家的挖掘速度。",
+	"enchantment.wonderful_enchantments.incompatibility_curse": "不兼容诅咒",
+	"enchantment.wonderful_enchantments.incompatibility_curse.desc": "使该物品无法附魔。",
+	"enchantment.wonderful_enchantments.incompatibility_curse.description": "使该物品无法附魔。",
+	"enchantment.wonderful_enchantments.vampirism_curse": "吸血鬼诅咒",
+	"enchantment.wonderful_enchantments.vampirism_curse.desc": "当玩家在阳光下时变得虚弱且着火，但会增强蛭吸附魔。",
+	"enchantment.wonderful_enchantments.vampirism_curse.description": "当玩家在阳光下时变得虚弱且着火，但会增强蛭吸附魔。",
+	"enchantment.wonderful_enchantments.corrosion_curse": "侵蚀诅咒",
+	"enchantment.wonderful_enchantments.corrosion_curse.desc": "当与水接触时，逐渐破坏盔甲并伤害所有者。",
+	"enchantment.wonderful_enchantments.corrosion_curse.description": "当与水接触时，逐渐破坏盔甲并伤害所有者。",
+	"item.wonderful_enchantments.wonderful_book": "奇异之书",
+	"item.wonderful_enchantments.wonderful_book.enchanting_energy": "附魔能量：",
+	"item.wonderful_enchantments.wonderful_book.enchanting_energy_max": "最大附魔能量！",
+	"item.wonderful_enchantments.wonderful_book.enchanting_cost": "花费等级：",
+	"item.wonderful_enchantments.wonderful_book.level": "级",
+	"item.wonderful_enchantments.wonderful_book.levels": "级",
+	"item.wonderful_enchantments.wonderful_book.hint": "你可以在合成栏上用青金石增加附魔能量。",
+	"item.wonderful_enchantments.wonderful_book.transmute": "手持并右击以将其转化为附魔书\ue908\ue909"

```

这里请直接用省略号。用转义符号`\ue908\ue909`可能会在游戏内显示错误
```suggestion
	"item.wonderful_enchantments.wonderful_book.transmute": "手持并右击以将其转化为附魔书……"
```

---

这个是原翻译 那我改吧